contexts/did-resolution-v1.json

```diff
@@ -1,22 +1,44 @@
 {
-  "@context": [
-    {
-      "@version": 1.1
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "didres": "https://w3id.org/did-resolution#",
+
+    "didDocument": "didres:didDocument",
```

Comment:
can we fully qualify rather than use the short-form? That's becoming a standard we've started using elsewhere that's worked pretty well.

Reply:
Yes sounds good.